studio/components/ui/ConsentToast.tsx

```diff
@@ -0,0 +1,38 @@
+import { noop } from 'lodash'
+import Link from 'next/link'
+import { Button } from 'ui'
+
+interface ConsentToastProps {
+  onAccept: () => void
+  onOptOut: () => void
+}
+
+const ConsentToast = ({ onAccept = noop, onOptOut = noop }: ConsentToastProps) => {
+  return (
+    <div className="space-y-3">
+      <div className="text-sm">
+        <p>
+          <span className="underline font-medium">No cookies.</span> 🍪
+        </p>
+        <p>
+          We only collect analytics essential to ensuring smooth operation of our services.
+        </p>
+      </div>
+      <div className="flex items-center space-x-2">
+        <Button type="default" onClick={onAccept}>
+          Accept
+        </Button>
+        <Button type="text" onClick={onOptOut}>
+          Opt out
+        </Button>
+        <Link href="https://supabase.com/privacy">
```

Comment:
```
<Link passHref target="_blank" rel="noreferrer">
  <Button asChild type="text">
    <a>Learn More 
    //..
```

Reply:
removed this file instead and use the one from ui